### ISSUE TITLE
Light Religion Changes

### ISSUE THREAD

**Issue by Grekopithikos** [Warcraft-GoA-Development-Team/Warcraft-Guardians-of-Azeroth-2#827]
**Describe your suggestion in full detail below:**

I would like to propose a few changes in the Light faiths.

The first one would be adding Stratholme as one of the Holy Light holy sites, since it is one in the lore according to https://wowpedia.fandom.com/wiki/Stratholme.
To quote the link, "Among the oldest and most revered of these holy sites were the temples in the Eastweald, including Stratholme." and "It was also a holy site, being the Silver Hand paladin order's birthplace.".
I think CK3 does allow for more than 5 holy sites, but if you really need to swap it with another holy site, I would suggest swapping it with the holy site of Ironforge, as I can't really see why it is a holy site in the first place, since the dwarves now have their own Light faith, Khazism, making the main Holy Light faith more human centered.
I can only assume that's a leftover from the CK2 mod where the dwarves and the humans had the same religion.

The other change I'd like to propose would be to make order or arcane magic accepted in the Scarlet Light faith, since they do have mages in their order, the most notable being https://wowpedia.fandom.com/wiki/Arcanist_Doan and https://wowpedia.fandom.com/wiki/Flameweaver_Koegler.
In fact I'd even go as far as to have them accept spirit magic, or whichever doctrine will allow for the monk class, as I remember there being monks in the Scarlet Monastery dungeon(https://wowpedia.fandom.com/wiki/Scarlet_Monk).

My final suggestion would be the implementation of an Argent Dawn faith.
I already know that this faith is planned, but I'd like to put some input in the design.
As they accept Undead into their ranks, they should have a doctrine similar to the Tauren faith Earth Mother's Mercy, where being undead isn't a crime in and of itself.
As for the tenets, I'd suggest Armed Pilgrimages, since they are a crusade, Adaptive, since they even accept members of Horde races into their ranks, and maybe Unrelenting Faith, since they are militant.
As for the holy sites they'd be the same as the ones of Holy Light plus Icecrown.

**Comment by Grekopithikos** [comment]
Here's a better example of a Scarlet Light Monk https://wowpedia.fandom.com/wiki/Brother_Korloff

**Comment by Loralius** [comment]
I agree with removing Ironforge as a holy site for the Holy Light _in light_ of the Khazism faith. You make a strong case for the inclusion of Stratholme in its stead.

**Comment by Grekopithikos** [comment]
I suppose now with #765's update, it would now be fire magic controlled, on top of arcane being accepted.

**Comment by Loralius** [comment]
These changes make sense to me too, particularly after consulting the _Rank and Classes_ section for the [Scarlet Crusade on WoWpedia](https://wowpedia.fandom.com/wiki/Scarlet_Crusade#Ranks_and_classes) and looking into the lore behind non-sanctioned Kirin Tor self-taught human hedge wizards and sorcerors. Given the fundamentalist zealotry of the Scarlet Crusade and their preoccupation with unrelenting purification, I think it wouldn't be unreasonable to set their attitude towards controlled fire magic as idolised.
